qiskit/algorithms/amplitude_estimators/ae_utils.py

```diff
@@ -39,20 +39,20 @@ def bisect_max(f, a, b, steps=50, minwidth=1e-12, retval=False):
     m = (a + b) / 2
     fm = 0
     while it < steps and b - a > minwidth:
-        l, r = (a + m) / 2, (m + b) / 2
-        fl, fm, fr = f(l), f(m), f(r)
+        left, r = (a + m) / 2, (m + b) / 2
+        fl, fm, fr = f(left), f(m), f(r)
```

Comment:
I agree. I think we've discussed this at least once in the past. For example, it can be much easier to reference an algorithm in a paper if you use the same symbols in the code. And the exceptions add clutter. I also don't feel strongly about whether to use a checker. Having said that, I often find that names are poorly chosen, and can make code difficult to read and search (Could use a better IDE in this case, I suppose)

Reply:
Fixed (reverted the commits) in https://github.com/Qiskit/qiskit-terra/pull/9586#commits-pushed-0662c05